### ISSUE TITLE
Switching map to another browser window

### ISSUE THREAD

**Issue by andrewcoder002** [openlayers/openlayers#12888]
I would be nice to have possibility to switch already inicialized map to another browser window. This window can user place on 2nd display ane leave original page to other application functionality (tables, graphs, forms).
It can be partialy achieved by opening new browser window and calling _map.setTarget_ method, but new map has some event handlers (mouse move, window resize,...) still bounded to original window. Can you consider modifying _handleTargetChanged_()_ method to correctly cover this situation?

**Comment by andrewcoder002** [comment]
I still use old ol.js script in my projects. Could you help me how to build it from source? I tried to use _build-legacy_ script, but it crashes with some errors:

```
\openlayers6.8.1\tasks\prepare-package.js:1
import esMain from 'es-main';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Module._compile (internal/modules/cjs/loader.js:895:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10)
    at internal/main/run_main_module.js:17:11
```
Can you pls help me to put the build process into operation?

**Comment by tschaub** [comment]
Fixed with #12893.
